### ISSUE TITLE
Issue list before functional test could be merged

### ISSUE THREAD

**Issue by Crazykev** [hyperhq/runv#481]
Track issue list in runv before functional test #474 could pass and get merged.
- BUG
  - [ ]  detach mode(-d) in run/create/start works abnormally. 
- Improvement
  - [ ] #472 Link to vm root for test improvement.
  - [ ] Error message format.

**Comment by gnawux** [comment]
Any update on this issue @Crazykev ?

**Comment by Crazykev** [comment]
Nope, Not sure where runv should go at this point, there are still many things need to do to keep up with current containerd and OCI runtime spec.
